<UI version="4.0" >
 <class>ProperTrenchCoatShopClass</class>
 <widget class="QMainWindow" name="ProperTrenchCoatShopClass" >
  <property name="objectName" >
   <string notr="true">ProperTrenchCoatShopClass</string>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
 <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QListWidget" name="movieListWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="saveToMyListButton">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="myListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="IDLabel">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="IDLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="sizeLabel">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="sizeLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="colourLabel">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Colour</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="colourLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="priceLabel">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Price</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="priceLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="quantityLabel">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Quantity</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="quantityLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
			<item row="5" column="0">
             <widget class="QLabel" name="photographyUrlLabel">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Photography</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="photographyUrlLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="redoButton">
            <property name="text">
             <string>Redo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="titleLabel_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>FileLocation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="fileLocationLineEdit">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="updateFileLocationButton">
            <property name="text">
             <string>Update File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="userButton">
          <property name="text">
           <string>Mode B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adminButton">
          <property name="text">
           <string>Mode A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Sort by Genre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sizeToSortByLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="nextButton">
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sortBySizeButton">
              <property name="text">
               <string>Sort</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="1">
             <widget class="QLineEdit" name="fileLocationLineEdit_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="titleLabel_3">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>MyList Location</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="updateMyListLocationButton">
              <property name="text">
               <string>Update File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="openWatchListButton">
              <property name="text">
               <string>Open Watch List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="openMyListButton">
              <property name="text">
               <string>Open My List In Window</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</UI>
